exchanges/exmo/exmo_wrapper.go

```diff
@@ -203,35 +201,45 @@ func (e *EXMO) UpdateTradablePairs(ctx context.Context, forceUpdate bool) error
 
 // UpdateTickers updates the ticker for all currency pairs of a given asset type
 func (e *EXMO) UpdateTickers(ctx context.Context, a asset.Item) error {
-	result, err := e.GetTicker(ctx)
+	avail, err := e.GetAvailablePairs(a)
 	if err != nil {
 		return err
 	}
-	pairs, err := e.GetEnabledPairs(a)
+
+	enabled, err := e.GetEnabledPairs(a)
 	if err != nil {
 		return err
 	}
-	for i := range pairs {
-		for j := range result {
-			if !strings.EqualFold(pairs[i].String(), j) {
-				continue
-			}
 
-			err = ticker.ProcessTicker(&ticker.Price{
-				Pair:         pairs[i],
-				Last:         result[j].Last,
-				Ask:          result[j].Sell,
-				High:         result[j].High,
-				Bid:          result[j].Buy,
-				Low:          result[j].Low,
-				Volume:       result[j].Volume,
-				ExchangeName: e.Name,
-				AssetType:    a})
-			if err != nil {
-				return err
-			}
+	result, err := e.GetTicker(ctx)
+	if err != nil {
+		return err
+	}
+
+	for symbol, tick := range result {
+		pair, err := avail.DeriveFrom(strings.Replace(symbol, "_", "", 1))
```

Comment:
```suggestion
		var pair currency.Pair
		pair, err = avail.DeriveFrom(strings.Replace(symbol, "_", "", 1))
```
I look forward to your updated `DeriveFrom` func